ui/apps/platform/src/Containers/NetworkGraph/TopologyComponent.tsx

```diff
@@ -115,14 +116,6 @@ export type TopologyComponentProps = {
     scopeHierarchy: NetworkScopeHierarchy;
 };
 
-// @TODO: Consider a better approach to managing the side panel related state (simulation + URL path for entities)
-function clearSimulationQuery(search: string): string {
-    const modifiedSearchFilter = getQueryObject(search);
-    delete modifiedSearchFilter.simulation;
-    const queryString = getQueryString(modifiedSearchFilter);
-    return queryString;
-}
-
```

Comment:
Moving this to the NetworkPolicySimulatorSidePanel and making it exports for cleaner reuse higher in the component tree.